website/content/docs/ecs/manual-installation.mdx

```diff
@@ -0,0 +1,507 @@
+
+Refer to the [Consul Agent documentation](/docs/agent/options#configuration_files) for a complete reference of Consul agent
+configuration options.
+
+## `mesh-init` container
+
+The `mesh-init` container runs at task startup to setup this instance for Consul service mesh.
+It registers the service and proxy for this task with Consul and writes Envoy bootstrap
+configuration to a shared volume.
+
+<CodeBlockConfig highlight="18-41">
+
+```json
+{
+  "containerDefinitions": [
+    {
+      "name": "example-client-app",
+      "image": "docker.io/org/my_task:v0.0.1",
+      ...
+    },
+    {
+      "name": "sidecar-proxy",
+      "image": "envoyproxy/envoy-alpine:<ENVOY_VERSION>",
+      ...
+    },
+    {
+      "name": "consul-client"
+      "image": "public.ecr.aws/hashicorp/consul:<CONSUL_VERSION>",
+      ...
+    },
+    {
+      "name": "consul-ecs-mesh-init",
+      "image": "public.ecr.aws/hashicorp/consul-ecs:<CONSUL_ECS_VERSION>",
+      "command": ["mesh-init"],
+      "essential": false,
+      "environment": [
+        {
+          "name": "CONSUL_ECS_CONFIG_JSON",
+          "value": "{\"bootstrapDir\":\"/consul\",\"healthSyncContainers\":[],\"proxy\":{\"upstreams\":[{\"destinationName\":\"example-server-app\",\"localBindPort\":1234}]},\"service\":{\"checks\":[],\"meta\":{},\"name\":\"example-client-app\",\"port\":9090,\"tags\":[]}}"
+        }
+      ],
+      "mountPoints": [
+        {
+          "readOnly": false,
+          "containerPath": "/consul",
+          "sourceVolume": "consul_data"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/bin/consul-inject",
+          "sourceVolume": "consul_binary"
+        }
+      ]
+    }
+  ]
+}
+```
+
+</CodeBlockConfig>
+
+| Field name    | Type    | Description                                                                                                         |
+| -----------   | ------- | ------------------------------------------------------------------------------------------------------------------- |
+| `name`        | string  | The container name should be `consul-ecs-mesh-init`.                                                                |
+| `image`       | string  | The `consul-ecs` image. Use our public AWS registry, `public.ecr.aws/hashicorp/consul-ecs`, to avoid rate limits.   |
+| `mountPoints` | list    | Must be set as show above, so the `consul` and `consul-ecs` binaries can be shared among containers for task setup. |
+| `command`     | list    | Set the `["mesh-init"]` so that the container runs the `consul-ecs mesh-init` command.                              |
+| `environment` | list    | This must include the `CONSUL_ECS_CONFIG_JSON` variable. See below for details.                                     |
+
+Configuration is passed to the `consul-ecs mesh-init` command in JSON format using the `CONSUL_ECS_CONFIG_JSON` environment variable.
+Here is the sample config from above, expanded to be readable:
```

Comment:
```suggestion
Here is the sample config from above, expanded to be readable:
```
Which configuration from above? Maybe there should be a section called "Example Configuration" at the end of the topic that we can refer to?

Reply:
I didn't add a new section for reference documentation, but I did reword things in this section to clarify a little. See what you think!